src/server/auth/testing/auth.go

```diff
@@ -27,6 +27,11 @@ func (a *InactiveAPIServer) Authorize(ctx context.Context, req *auth.AuthorizeRe
 	return nil, auth.NotActivatedError{}
 }
 
+// WhoAmI implements the Pachdyerm Auth WhoAmI RPC, but just returns NotAuthorizedError
```

Comment:
It seems to actually return `NotActivatedError` so either the comment or the implementation should be changed.